builds/jazz-build-module/azure-deploy-module.groovy

```diff
@@ -0,0 +1,64 @@
+#!groovy?
+
+/*
+* azure deployment module
+*/
+
+
+echo "azure deployment module loaded successfully"
+
+def createFunction(stackName){
+
+  sh "az functionapp create -s heinajazzdevsa -n $stackName --consumption-plan-location westus2 >> output.log"
```

Comment:
@oscarbrooksdev9  sorry maybe here is the issue
Please use your storage account which match with your default resource group.
heinajazzdevsa wont work for you.